challenge_4/sass/main.scss

```diff
@@ -0,0 +1,60 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+$primary-color: #998235;
+$secondary-color: #0B2429;
+
+.container {
+  background: $secondary-color;
+  height: 300px;
+  width: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 60px;
+}
+
+.left-triangle {
+  height: 55px;
+  border-top: 50px solid transparent;
+  border-bottom: 50px solid transparent;
+  border-left: 48px solid $primary-color;
+}
+
+.right-triangle {
+  height: 55px;
+    border-top: 50px solid transparent;
+    border-bottom: 50px solid transparent;
+    border-right: 48px solid #998235;
+    margin-right: -14px;
+}
+
+.big-circle {
+  background: #998235;
+  border-radius: 50%;
+  height: 200px;
+  margin-right: -14px;
+  display: flex;
+  width: 200px;
+  align-items: center;
+  padding: 12px;
+}
+
+.middle-circle {
+  width: 213px;
+  height: 178px;
+  background-color: #0B2429;
+  border-radius: 53%;
+  padding: 18px;
+  z-index: 1;
+}
+
+.smallest-circle {
+  border: 46px solid #F3AC3C;
+  border-radius: 50%;
+  width: 140px;
+  height: 140px;
```

Comment:
- [ ] You can also nest these circles: 

```css
.circle {
  &-big {...}
  &-middle {...}
  &-smallest {...}
}
```

- [ ] And please use variables for the colors